src/openrct2-ui/windows/EditorObjectSelection.cpp

```diff
@@ -957,10 +957,8 @@ class EditorObjectSelectionWindow final : public Window
         }
 
         const int32_t ride_tabs[] = {
-            SPR_TAB_RIDE_16,        IMAGE_TYPE_REMAP | SPR_TAB_RIDES_TRANSPORT_0,
-            SPR_TAB_RIDES_GENTLE_0, IMAGE_TYPE_REMAP | SPR_TAB_RIDES_ROLLER_COASTERS_0,
-            SPR_TAB_RIDES_THRILL_0, SPR_TAB_RIDES_WATER_0,
-            SPR_TAB_RIDES_SHOP_0,   SPR_TAB_FINANCES_RESEARCH_0,
+            SPR_TAB_RIDE_16,        SPR_TAB_RIDES_TRANSPORT_0, SPR_TAB_RIDES_GENTLE_0, SPR_TAB_RIDES_ROLLER_COASTERS_0,
+            SPR_TAB_RIDES_THRILL_0, SPR_TAB_RIDES_WATER_0,     SPR_TAB_RIDES_SHOP_0,   SPR_TAB_FINANCES_RESEARCH_0,
```

Comment:
Test note: do these tab images still work correctly?

Reply:
My changes to ImageId stopped them from working. Because it was using the constructor for ImageId which only expects the index, not an original uint32 of the image. If you use the constructor where you pass the colour, that implies remap. So this fixes them.